lib/bug_report_templates/rails_5.rb

```diff
@@ -0,0 +1,109 @@
+begin
+  require 'bundler/inline'
+rescue LoadError => e
+  STDERR.puts 'Bundler version 1.10 or later is required. Please update your Bundler'
+  raise e
+end
+
+gemfile(true) do
+  source 'https://rubygems.org'
+
+  gem 'rails', require: false
+  gem 'sqlite3'
+  gem 'activeadmin', git: 'https://github.com/activeadmin/activeadmin', require: false
+  gem 'inherited_resources', '~> 1.7', git: 'https://github.com/activeadmin/inherited_resources', require: false
```

Comment:
I don't think we need to point to master here. Actually, we should update the equivalent line in README and Rails 5.0 gemfile as well. But we can do that in a separate PR.